docs-ref-conceptual/eclipse/azure-toolkit-for-eclipse-create-hello-world-web-app.md

```diff
@@ -71,7 +95,7 @@ When you have completed this tutorial, your application will look similar to the
 
 8. Save index.jsp.
 
-## Deploy your web app to Azure
+## Deploying web app to Azure
```

Comment:
Some titles used present continuous tense and some used present tense. Do you want to unify the formats? Thanks.

Creating web app project
Deploying web app to Azure
Display the Azure Explorer view
Cleaning up resources

Reply:
done fixing in the latest commit